uniffi_bindgen/src/scaffolding/templates/ExternalTypesTemplate.rs

```diff
@@ -8,6 +8,8 @@
 ::uniffi::ffi_converter_forward!(r#{{ name }}, ::{{ crate_name|crate_name_rs }}::UniFfiTag, crate::UniFfiTag);
 {%- when ExternalKind::Interface %}
 ::uniffi::ffi_converter_forward!(::std::sync::Arc<r#{{ name }}>, ::{{ crate_name|crate_name_rs }}::UniFfiTag, crate::UniFfiTag);
+{%- when ExternalKind::CallbackInterface %}
+::uniffi::ffi_converter_forward!(Box<dyn r#{{ name }}>, ::{{ crate_name|crate_name_rs }}::UniFfiTag, crate::UniFfiTag);
```

Comment:
I dont quite understand the difference between this and uniffi_bindgen/src/scaffolding/mod.rs but I've just tried to do the same thing